keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,339 @@
+in the docs provide sufficient value to the reader that they should remain?
+
+Casual consensus says yes, with one modification:
+- Eliminate the [production environment table](https://kubernetes.io/docs/setup/#production-environment)
+with a link to [certified conformance partners](https://kubernetes.io/partners/#conformance).
+
+1. Is there a list of content pages that are so focused on third party product
+usage that they should be removed or updated from the Kubernetes documentation?
+
+See https://github.com/kubernetes/website/issues/15748.
+
+1. When should the Kubernetes documentation host third party content that isn't
+maintained by a Kubernetes SIG?
+
+As infrequently as possible, with linking preferred to hosting. 
+
+1. How does the Kubernetes project handle third party content that is not kept
+up to date or hosts?
+
+If content isn't refreshed within 180 days, notify stakeholders of 90 days to 
+update content or migrate it elsewhere before removing it.
+
+1. Can feature owners flag when third party content is *required*, as opposed to
+preferable or common?
+
+Is this capability required for KEP approval?
+
+1. Who decides when to include third-party content?
+
+SIGs responsible for particular features can include third-party content at
+their discretion, preferably by linking to the third party's own documentation.
+
+1. What standard of quality and review must be met before docs include
+third-party content?
+
+Third-party content must be necessary for Kubernetes to function in-project.
+
+1. To what extent should SIG Docs advocate for third-party content providers to 
+host their own content, or decline to host third-party content altogether?
+
+Kubernetes docs publish third-party content only if:
+
+- It's necessary for Kubernetes to function. For example: container runtimes 
+(containerd, CRI-o, Docker), networking policy (CNI plugins), Ingress 
+controllers, and logging.
+
+- It's an applied example of another project in the Kubernetes GitHub org. This
+includes the [kubernetes](https://github.com/kubernetes) and 
+[kubernetes-sigs](https://github.com/kubernetes-sigs) repositories.
+
+Third-party content should be linked instead of hosted whenever possible. 
+
+</del>
+
+1. Clearly define what documentation is required so that readers understand
+   how to deploy, operate and consume Kubernetes clusters using features from
+   in-project code and its mandatory dependencies.
+
+### Non-Goals
+
+1. Outright removal of all content relating to vendors and projects
+   outside the CNCF ecosystem.
+
+## Proposal
+
+1. Revise the [content guide](https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/style/content-guide.md#contributing-content) to achieve the KEP goal:
+
+- Specify that Kubernetes docs are limited to content required for Kubernetes to
+function in-project. Docs may include third-party content for components that 
```

Comment:
Does `third-party` include open source projects AND vendor specific? or just vendor specific? A definition like the `in-project` would be useful here

Reply:
@dims 
> Does third-party include open source projects AND vendor specific? or just vendor specific?

Ideally we'd be leaving this to the discretion of the relevant SIGs. In practice, allowing vendor-specific content created the race for inclusion we see now. 

Specifically, I don't want to deal with more angry Slack messages from vendors who can't get a mention in the same page as their competitor because their competitor has a contributor in the SIG and they don't.

Unless you have a better solution, let's limit third-party content to OSS projects that are necessary to run Kubernetes. To stem off inevitable what-aboutism re:Docker, can we acknowledge that Docker is an edge case without having to specifically say that Docker is the sole exception?